inginious/frontend/themes/Gruntfile.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
```

Comment:
You can rename `themes/themes` into `themes/src` to be less confusing.

You should probably add an initial comment such as 
   
    Based on Bootswatch from Thomas Park (https://github.com/thomaspark/bootswatch)
    
in the beginning of the imported files